### ISSUE TITLE
Go to anything relies on cache too much

### ISSUE THREAD

**Issue by vsespb** [Komodo/KomodoEdit#1018]
In the screenshot below there is a project with file t2.pl, while Ctlr-Shift-O cannot find "t2.pl".

Reproduce: add new files while Komodo running, try to find it.
Reproducible: not always

Note: If can't reproduce try to remove some files, which are open in Komodo right now + add some new.

Happens often, annoying. Usually happens when swicth git branches on open project.

Komodo Edit, version 9.3.2, build 16460, platform linux-x86_64. Built on Thu Dec 10 19:37:22 2015.

![screenshot from 2016-01-21 01 19 34](https://cloud.githubusercontent.com/assets/1210230/12465137/a53b6350-bfdd-11e5-8488-7afe6ad3e851.png)

**Comment by Naatan** [comment]
Commando currently only catches new files that were added through Komodo. Did you add this file by other means by any chance?

**Comment by vsespb** [comment]
Yes, I added without using Komodo (usually happends when I switch git branches via command line).

> Commando currently only catches new files that were added through Komodo.

1. Sometimes it *catches* files added not by Komodo
2. I think in Komodo-8 Ctrl-Shit-O was opening files fine...
3. So it there a hotkey to Open a file by file name (not File open dialog where you specify full path) including new files not added through Komodo ? Current behaviour just does not make sense..

**Comment by Naatan** [comment]
It can catch files if they reside in a folder that was not cached yet. You can force a clean cache by hitting the icon next to the search field and hitting "Clear Cache".

**Comment by Naatan** [comment]
Closing this as its a known limitation. It will be addressed but it is not a bug.

**Comment by vsespb** [comment]
"Clear Cache" does not help

**Comment by Naatan** [comment]
I believe you need to be inside the Files scope for it to work.

**Comment by vsespb** [comment]
ok, seems works. thx

**Comment by felipsmartins** [comment]
"Clear cache" worked well for me.

**Comment by vsespb** [comment]
The need to switch to files, and to clear cache seems show stopper for me. Very annoying. Have to use it very often

**Comment by Naatan** [comment]
Repurposing this big for addressing the issue with caching.

**Comment by vsespb** [comment]
thx!
